core/templates/pages/exploration-player-page/layout-directives/learner-view-info.directive.ts

```diff
@@ -98,6 +131,32 @@ angular.module('oppia').directive('learnerViewInfo', [
                 ctrl.explorationTitle = response.exploration.title;
                 $rootScope.$applyAsync();
               });
+            // To check if the exploration is linked to the topic or not.
+            ctrl.isLinkedToTopic =
```

Comment:
Use parenthesis when you want to break a statement (see [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#javascript)).